service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandlerTest.java

```diff
@@ -145,7 +159,38 @@ void shouldNotifyNonHmctsAdminOnC2Upload() throws IOException, NotificationClien
             notificationHandler.sendNotificationForC2Upload(new C2UploadedEvent(callbackRequest(), AUTH_TOKEN, USER_ID));
 
             verify(notificationClient, times(1)).sendEmail(
-                eq(C2_UPLOAD_NOTIFICATION_TEMPLATE), eq("hmcts-non-admin@test.com"), eq(parameters), eq("12345"));
+                eq(C2_UPLOAD_NOTIFICATION_TEMPLATE.getTemplateId()), eq("hmcts-non-admin@test.com"), eq(parameters), eq("12345"));
+        }
+
+        @Test
+        void shouldNotifyPartiesOnC21OrderSubmission() throws IOException, NotificationClientException {
+            final LocalDate hearingDate = LocalDate.now().plusMonths(4);
+
+            given(dateFormatterService.formatLocalDateToString(hearingDate, FormatStyle.MEDIUM))
+                .willReturn(hearingDate.format(
+                    DateTimeFormatter.ofLocalizedDate(FormatStyle.MEDIUM).localizedBy(Locale.UK)));
+
+            given(hmctsCourtLookupConfiguration.getCourt(LOCAL_AUTHORITY_CODE))
+                .willReturn(new Court(COURT_NAME, COURT_EMAIL_ADDRESS));
+
+            given(cafcassLookupConfiguration.getCafcass(LOCAL_AUTHORITY_CODE))
+                .willReturn(new Cafcass(CAFCASS_NAME, CAFCASS_EMAIL_ADDRESS));
+
+            given(localAuthorityNameLookupConfiguration.getLocalAuthorityName(LOCAL_AUTHORITY_CODE))
+                .willReturn("Example Local Authority");
+
+            given(c21OrderEmailContentProvider.buildC21OrderNotification(callbackRequest().getCaseDetails(),
+                LOCAL_AUTHORITY_CODE)).willReturn(parameters);
```

Comment:
can all of this be moved outside of the test. Lots of clutter. I only care that an email was sent to a party in this test